tests/test_krr.py

```
@@ -5,14 +5,29 @@
 
 from robusta_krr.main import app, load_commands
 from robusta_krr.core.integrations.kubernetes import ClusterLoader
-from robusta_krr.core.models.config import settings
+from robusta_krr.core.models.config import settings, Config
```

Flake8 (7.3.0)

[error] 'robusta_krr.core.models.config.settings' imported but unused

(F401)